jdi-light-angular-tests/src/test/java/io/github/epam/TestsInit.java

```diff
@@ -0,0 +1,40 @@
+package io.github.epam;
+
+
+import static com.epam.jdi.light.driver.WebDriverUtils.killAllSeleniumDrivers;
+import static com.epam.jdi.light.driver.get.DriverData.DRIVER_NAME;
+import static com.epam.jdi.light.elements.composite.WebPage.openUrl;
+import static com.epam.jdi.light.elements.init.PageFactory.initSite;
+import static com.epam.jdi.light.settings.WebSettings.DOMAIN;
+import static com.epam.jdi.light.settings.WebSettings.logger;
+
+import com.epam.jdi.light.settings.WebSettings;
+import io.github.com.StaticSite;
+import io.github.epam.testng.TestNGListener;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Listeners;
+
+
+@SuppressWarnings("PMD.ClassNamingConventions")
```

Comment:
remove suppression, do we need it?

Reply:
Yes, we need it because this class is recognized by Codacy as utility class (it has only static methods), and its name should end with "Utils" or "Helper"

Reply:
Removed